src/NUnitFramework/framework/Internal/Execution/TextMessageWriter.cs

```diff
@@ -37,7 +37,7 @@ namespace NUnit.Framework.Internal
     public class TextMessageWriter : MessageWriter
     {
         #region Message Formats and Constants
-        private static readonly int DEFAULT_LINE_LENGTH = 78;
+        private static int Default_Line_Length = 78;
```

Comment:
Done.  FYI - my intention in this patch is to achieve something similar to the maxDiff property of python's unittest framework (https://docs.python.org/2.7/library/unittest.html).  I admit that there may be a cleaner way to achieve this.